### ISSUE TITLE
Reference highlighting when jumped to from citations on web version of article

### ISSUE THREAD

**Issue by severett89** [openlibhums/janeway#4337]
**Is your feature request related to a problem? Please describe.**
At the moment when citations are clicked in the body text of an article, the webpage jumps to the bottom where the references hang out. At the moment this positions the selected reference at the top of the web browser window, however for references towards the bottom of the webpage they cannot reach the top of the browser as the webpage can't scroll any further down.

**Describe the solution you'd like**
Potentially highlighting selected references somehow when they are jumped to (or invoking some kind of visual cue) so that the user knows which reference they should be looking at if it cannot be positioned as the topmost item in the browser window.

**Comment by StephDriver** [comment]
see also 
- #3242 
- #3939

**Comment by StephDriver** [comment]
Backlog refinement - this needs to be considered along with other A11y updates to in page links. @StephDriver to investigate and group them.

**Comment by StephDriver** [comment]
branch has been merged into master
